_data/team/roles/non_core_library_maintenance.yml

```diff
@@ -13,52 +13,53 @@ subroles:
 
   - subrole: distopia
     current_members:
-      - Richard Gowers
-      - Hugo MacDermott-Opeskin
-      - Rocco Meli
+      - "[Richard Gowers](https://github.com/richardjgowers)"
+      - "[Hugo MacDermott-Opeskin](https://github.com/hmacdope)"
+      - "[Rocco Meli](https://github.com/RMeli)"
```

Comment:
```suggestion
```
I haven't done anything with `dystopia` in the end. =( I should be removed here.